File: ad-hoc/2388/2388.c
// 2388 - Tacografo - beecrowd

#include <stdio.h>

void calc(int N) 
{
    int t, v, s = 0;
 
    for (int i = 0; i < N; i++)
    {
        scanf("%d %d", &t, &v);
        s += t * v;
    }

    printf("%d\n", s);
}

int main(void)
{
    int n; scanf("%d", &n);
    
    calc(n);

    return 0;
}

File: ad-hoc/2388/2388.py
# 2388 - Tacografo - beecrowd

def calc(n : int):
    s = 0

    for _ in range(n):
        t, v = map(int, input().split())
        s += t * v
    
    print(s)

def main():
    n = int(input())
    calc(n)

if __name__ == "__main__":
    main()
